sdcm/tester.py

```diff
@@ -2287,13 +2287,27 @@ def _get_common_email_data(self) -> dict:
         start_time = format_timestamp(self.start_time)
         config_file_name = ";".join(os.path.splitext(os.path.basename(cfg))[0] for cfg in self.params["config_files"])
         test_status = self.get_test_status()
+        backend = self.params.get("cluster_backend")
+        backend = {"docker": "Docker"}.get(backend, backend.upper())
+        if backend == 'AWS':
+            region_name = self.params.get('region_name')
+        elif backend == 'GCE':
+            region_name = self.params.get('gce_datacenter')
+        else:
+            region_name = ''
+        build_id = os.environ.get('BUILD_NUMBER', '')
+        if build_id:
+            build_id = f' [{build_id}]'
         return {"build_url": os.environ.get("BUILD_URL"),
+                "build_id": os.environ.get('BUILD_NUMBER', ''),
                 "end_time": format_timestamp(time.time()),
                 "events_summary": self.get_event_summary(),
                 "last_events": self.get_events_grouped_by_category(100),
                 "nodes": [],
+                "backend": backend,
+                "region_name": region_name,
                 "start_time": start_time,
-                "subject": f"{test_status}: {os.environ.get('JOB_NAME') or config_file_name}: {start_time}",
+                "subject": f"{test_status}: {os.environ.get('JOB_NAME') or config_file_name}{build_id}: {start_time}",
```

Comment:
better separate jobname and build number using `#` for example `[qa] FAILED: scylla-4.2/longevity/longevity-200gb-48h-test#208: 2020-08-14 12:34:52`

Reply:
fixed